### DISCUSSION TITLE
how to get chain decimals

### DISCUSSION THREAD

**Discussion by ryanleecode** [polkadot-api/polkadot-api#652] [Q&A]
what is the equivalent of `api.registry.chainDecimals`?

**Comment by josepot** [comment]
As I'm sure that you already know: the decimals are not stored on-chain.

Therefore, there are only 2 reasonable ways to know the decimals:

1) If you know the chain that your DApps can connect beforehand, then you should create a configuration object with this kind of information (chain, token name, decimals, etc).
2) If you are creating some sort of dev-tool that wants to access this information dynamically, then you can try to obtain it from the `properties` field of the object returned by the `getChainSpecData()` function.
